rest/rest_collection_impl.h

```diff
@@ -82,6 +82,10 @@ struct RestCollection<Key, Value>::Impl {
         protected by the RCU lock.
     */
     MutateMutex mutateMutex; // not mutable on purpose
+
+    int runningThreads;
```

Comment:
I added a counter, but my knowledge of that class is insufficient to know whether there can be more than one or not.